util/package.json

```diff
@@ -39,6 +39,7 @@
 	"dependencies": {
 		"ajv": "^8.6.2",
 		"amqplib": "^0.8.0",
+		"chalk": "^4.1.2",
```

Comment:
Please switch to Nanocolors

https://github.com/ai/nanocolors

As is faster and uses less space

Thanks